extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -7,9 +7,55 @@ define('ext.wikia.adEngine.template.porvata', [
 	'ext.wikia.adEngine.video.player.ui.videoInterface',
 	'ext.wikia.adEngine.video.videoSettings',
 	'wikia.document',
+	'wikia.log',
+	'wikia.window',
 	require.optional('ext.wikia.adEngine.mobile.mercuryListener')
-], function (DOMElementTweaker, porvata, googleIma, uiTemplate, videoInterface, videoSettings, doc, mercuryListener) {
+], function (
+	DOMElementTweaker,
+	porvata,
+	googleIma,
+	uiTemplate,
+	videoInterface,
+	videoSettings,
+	doc,
+	win,
+	log,
+	mercuryListener
+) {
 	'use strict';
+	var logGroup = 'ext.wikia.adEngine.template.porvata';
+
+	function hideOtherBidsForVeles(params) {
+		if (params.adProduct === 'veles' && params.slotName === 'TOP_LEADERBOARD' && win.pbjs) {
```

Comment:
Correct me if I'm wrong, but `params.slotName` condition is not needed here - it should apply to all slots, right?